docs/developer-manual/deployments.md

```diff
@@ -0,0 +1,107 @@
+---
+id: orcfax deployments
+sidebar_position: 3
+slug: /deployments
+---
+
+# Protocol deployments
+
+Signposting current and historical deployments of the Orcfax protocol. The
+hashes can be used to navigate Orcfax fact-statements and archival packages
+on Cardano and Arweave blockchains.
+
+This document serves as a reference for historical access to Orcfax data.
+Integrators should consult the [integrator docs][consume] for an explanation
+an the role of different scripts used in the protocol and how to correctly use
+the Orcfax protocol.
+
+## V1
+
+Orcfax V1 introduces the FSP (fact statement pointer) script. The FSP will not
+change in the lifetime of a deployment. The FSP is used to record the current
+FS (fact statement) script hash. The FSP, FS, and C (constitution) script
+hashes are documented below for each V1 deployment.
+
+### V1: Mainnet
+
+A summary of mainnet deployments is listed below.
+
+| Status | TYPE | FSP | FS | Arweave Hash |
+|--------|------|---- |----|--------------|
+| TBD | CER | `TBD` | `TBD` | TBD |
+
+#### Mainnet detailed overview
+
+A per-FSP breakdown is documented below.
+
+|                  | Details |
+|------------------|---------|
+| FSP              | -       |
+| Arweave Hash     | -       |
+| First appearance | -       |
+| Slot no.         | -       |
+| Block            | -       |
+| Last appearance  | -       |
+| Slot no.         | -       |
+| Block            | -       |
+
+### V1: Preview
+
+A summary of preview deployments is listed below.
+
+| Status | TYPE | FSP | FS | C | Arweave Hash |
+|--------|------|---- |----|---|--------------|
+| Active | CER | `0690081bc113f74e04640ea78a87d88abbd2f18831c44c4064524230` | `572f979076428566452f5c22bc98f58d49f237d4f53b86d37b147244` | `3a81e444b7b88e41d421551d056ce1e7701948236251019d6fdce656` | [6KymaAPWd3JNyMT0B7EPYij4TWxehhMrzRD8qifCSLs][arweave-preview] |
+
+<br/>
+
+#### Preview detailed overview
```

Comment:
I had put V1.0 since it is reasonable to expect that we may have deployed v1 multiple time during testing. This _should_ not be the case for mainnet

Reply:
I don't disagree -- BUT -- we don't have a versioning scheme, and we don't have a second version, so it is a bridge to be crossed (that we really don't need to cross here)